app/views/green_lanes/measures/index.html.erb

```diff
@@ -0,0 +1,47 @@
+<h2>
+  Manage Green Lanes Measures
+</h2>
+
+<% if @measures.any? %>
+  <table>
+    <thead>
+    <tr>
+      <th>ID</th>
+      <th>Goods Nomenclature Item Id</th>
+      <th>Goods Nomenclature Description</th>
+      <th>Product Line Suffix</th>
+      <th>Theme</th>
+      <th>Measure Type</th>
+      <th>Regulation Id</th>
+    </tr>
+    </thead>
+    <tbody>
+    <% @measures.each do |me| %>
+      <tr id="<%= dom_id(me) %>">
+        <td><%= me.id%></td>
+        <td>
+          <%= me.goods_nomenclature.goods_nomenclature_item_id if me.goods_nomenclature.present?%>
+        </td>
+        <td>
+          <%= me.goods_nomenclature.description if me.goods_nomenclature.present?%>
+        </td>
+        <td><%= me.productline_suffix %></td>
+        <td>
+          <%= me.category_assessment&.theme&.code if me.category_assessment.present? && category_assessment.theme.present?%>
```

Comment:
`if` condition is superfluous - safe navigation operator measns its not needed - also the second condition looks wrong?